src/containers/MainContainer.jsx

```diff
@@ -17,13 +24,14 @@ class MainContainer extends Component {
     scaleY: 1,
     selectedShapeName: '',
     draggable: false,
-    x: 100,
-    y: 100,
+    x: undefined,
```

Comment:
Using undefined here is as good as not adding the property or is it for a specific purpose, null might be better.